modules/compliance-operator-configure-scanning.adoc

```diff
@@ -10,7 +10,9 @@ Create a `ScanSettingBinding` object in the `openshift-compliance` namespace to
 
 [NOTE]
 ====
-This example uses `cis` and `cis-node` profiles, but {ocp} provides additional profiles. See "Understanding the Compliance Operator" in the "Additional resources" section for more information.
+* If you are using Compliance (2.0), you can schedule your scan manually instead of creating the `ScanSettingBinding` on the Compliance Operator.
```

Comment:
nit: Instead of `manually` do we want to say something to the effect of _directly through ACS using a Scan Configuration_?

Reply:
s/  If you are using Compliance (2.0), you can schedule your scan by using a scan configuration instead of creating the `ScanSettingBinding` on the Compliance Operator.

@rhmdnd @dashrews78 WDYT?

Reply:
s/ If you are using Compliance (2.0), you can schedule your scan by using ACS to create a compliance scan schedule instead of creating the `ScanSettingBinding` on the Compliance Operator.

And maybe add a link to that section??